flow360/component/simulation/meshing_param/edge_params.py

```diff
@@ -11,36 +11,82 @@
 
 
 class AngleBasedRefinement(Flow360BaseModel):
-    """Surface edge refinement by specifying curvature resolution angle."""
+    """
+    Surface edge refinement by specifying curvature resolution angle.
+
+    Example
+    -------
+
+      >>> fl.AngleBasedRefinement(value=8 * fl.u.deg)
```

Comment:
I would delete the spaces
```suggestion
      >>> fl.AngleBasedRefinement(value=8*fl.u.deg)
```